.github/workflows/build.yml

```diff
@@ -1,6 +1,12 @@
 name: Build
 
-on: [push]
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
@adispring I already addressed this in #3206. Would you mind abandoning that change?